pkg-builder.pl

```diff
@@ -190,7 +190,7 @@ ($)
       stage_fun  => sub { &stage_zimbra_common_mbox_docs(@_); },
    },
    
-   "zimbra-core-jar" => {
+   "zimbra-common-core-jar" => {
```

Comment:
Minor: good if you change name of staging function

Reply:
done